deploy/application/main.tf

```
@@ -31,3 +31,12 @@ resource "helm_release" "this" {
     })
   ]
 }
+
+data "aws_ssm_parameters_by_path" "platform" {
+  path = "/platform/" 
```

```suggestion
  path = "/platform/${var.eks_name}*" 
```
with eks_name a variable with default to ""

---

(to test, I don't know if wildcards work or if path="/platform/xyz" will return all "/platform/xyz-123", "/platform/xyz-456", ...)

---

It's not working :-(